site/components/pages/Pledge/PledgeDashboard/Cards/AssetBalanceCard/index.tsx

```diff
@@ -64,11 +66,19 @@ export const AssetBalanceCard: FC<Props> = (props) => {
   const [balances, setBalances] = useState<Balances | null>(null);
 
   const holdingsByToken = groupBy(props.holdings, "token");
+
+  const emptyBalanceText = t({
+    id: "pledges.dashboard.assets.emptyBalance",
+    message: "No carbon assets to show.",
+  });
```

Comment:
This can be written below as:
```ts
import { Trans } from `lingui/macro`;

<Text t="body2">
  <Trans id="pledges.dashboard.assets.emptyBalance">
    No carbon assets to show.
  </Trans>
</Text>
```

Reply:
Much cleaner👍